packages/blade/src/components/ChatBubble/ChatBubble.web.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { SelfMessageBubble } from './SelfMessageBubble';
+import { DefaultMessageBubble } from './DefaultMessageBubble';
+import type { ChatBubbleProps } from './types';
+import { Text } from '~components/Typography';
+import BaseBox from '~components/Box/BaseBox';
+import { getStringFromReactText } from '~utils/getStringChildren';
+import { assignWithoutSideEffects } from '~utils/assignWithoutSideEffects';
+import type { BladeElementRef } from '~utils/types';
+
+const _ChatBubble = (
```

Comment:
Are the types correct? Let's follow the same convention here like we do in other places: 

React.ForwardRefRenderFunction<BladeElementRef, ChatBubbleProps>